platforms/software/dependency-management/src/main/java/org/gradle/api/internal/artifacts/ivyservice/resolveengine/graph/builder/DependencyGraphBuilder.java

```diff
@@ -207,9 +204,7 @@ private void traverseGraph(final ResolveState resolveState) {
                 // Initialize and collect any new outgoing edges of this node
                 dependencies.clear();
                 node.visitOutgoingDependencies(dependencies);
```

Comment:
💅 Since we are cleaning the logic here, maybe we could continue cleaning up names: `dependencies` is really `edges` - and it becomes that deeper down.
`visitOutgoingDependencies` could be `visitOutgoingDependenciesAndCollectEdges` - a mouthful bu that's what it does.